app/routes.js

```diff
@@ -1,6 +1,90 @@
-const express = require('express')
-const router = express.Router()
+const express = require("express");
+const HmppsOffenderAssessmentApi = require("hmpps_offender_assessment_api");
+const moment = require("moment");
+const axios = require("axios");
+
+const router = express.Router();
 
 // Add your routes here - above the module.exports line
 
-module.exports = router
+async function createAuthenticatedApiClient() {
+  const client = new HmppsOffenderAssessmentApi.ApiClient();
+
+  // For some reason the default basePath is https, which doesn't work
+  client.basePath = "http://localhost:8080";
+
+  const authTokenResponse = await axios.post(
+    "http://localhost:9090/auth/oauth/token?grant_type=client_credentials",
```

Comment:
Is it worth being explicit about where this is coming from in the PR description or readme?

Reply:
Good point - I've added a comment linking to that readme, and I'm doing the base64 conversion in the code instead of pasting it as a magic value.